### PR TITLE
Fix/setup

### PR DESCRIPTION
**Pull request by arthbous**
* added __version__ in  init
* remove automatic pytest
* remove fenics requirements

### CODE REVIEW

**Comment by maximilianmetti** [comment]
The code changes don't seem to reflect what the pr description is. Is this ready for review?

**Comment by arthbous** [comment]
Forgot to push...Now it reflects the description

**Comment by maximilianmetti** [comment]
Does this fix the issue that you were experiencing? Seems like you had an incompatibility w versions.  Shouldn't we set the install equal to what we have so the install gives a clearer error message "incompatible fenics version"?

**Comment by arthbous** [comment]
I added a warning in the version of Dolfin is wrong and the program stops if Dolfin is not installed. See `__init__.py`

File: README.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ def eafe_assemble(
   reac: Optional[dolfin.Expression] = None,
 ) -> dolfin.Matrix:
 ```
+which correspond to
+$$-\nabla\cdot(\alpha(x)\nabla(u) + \beta(x) * u) + R(x)u = f(x)$$
+* $\alpha(x)$ is the diffusion term
+* $\beta(x)$ is the diffusion term
```
Comment (maximilianmetti) [inline README.md]: convection

File: README.md [hunk 2]
```diff
@@ -18,6 +18,12 @@ def eafe_assemble(
...
+* $R(x)$ is the reaction term
```
Comment (maximilianmetti) [inline README.md]: If we use \alpha, \beta, then shouldn't we use \gamma for the reaction? Seems incosistent to switch to capital latin after we were using greek.

File: README.md [hunk 3]
```diff
@@ -18,6 +18,12 @@ def eafe_assemble(
   reac: Optional[dolfin.Expression] = None,
 ) -> dolfin.Matrix:
 ```
+which correspond to
```
Comment (maximilianmetti) [inline README.md]: corresponds

**Comment by arthbous** [comment]
Done. I also moved to HTML so it looks nicer on github

**Review by maximilianmetti — APPROVED** [review]

**Pull request merged**